onnxruntime/test/providers/cpu/math/matmul_integer_test.cc

```diff
@@ -32,6 +33,203 @@ TEST(MatmulIntegerOpTest, MatMulInteger) {
   test.AddOutput<int32_t>("T3", {1, 1}, {-1});
   test.Run();
 }
+
+// clang-format off
+TEST(MatmulIntegerOpTest, MatMulInteger_int8_t) {
+  OpTester test("MatMulInteger", 10);
```

Comment:
Do any of the tests validate this?
> The production MUST never overflow. The accumulation may overflow if and only if in 32 bits.

Reply:
It is guaranteed by our compute type. One int8 * int8 won't overflow with compute type int32.